0000-template.md

```diff
@@ -0,0 +1,26 @@
+# Feature Name
+
+## Summary
+[summary]: #summary
+
+One paragraph explanation of the feature.
+
+## Motivation
+[motivation]: #motivation
+
+Why are we doing this? What use cases does it support? What is the expected outcome?
+
+## Approach
+[approach]: #approach
+
+Provide a brief description of the implementation details for the feature. What
```

Comment:
Reflow to 80 cols? 

Or, as is usually best for prose: One sentence per line.

I am a bit torn here - this is filler text that should be removed, but the RFCs should be written as 1 sentence per line for easier diff reading.

Reply:
Yes, let's do the one sentence per line rule in the RFC. I have added a comment in the README.md about that.